src/quest_manager/templates/quest_manager/category_detail.html

```diff
@@ -0,0 +1,11 @@
+{% extends "quest_manager/base.html" %}
+{% load crispy_forms_tags %}
+
+{% block heading_inner %} {{ object.title }}{% endblock %}
```

Comment:
Seeing the result, please add "Campaign" after the title so it's more clear what this page is:
`{{ object.title }} Campaign`

Reply:
Also, add buttons to the  edit, and delete views, similar to the Quest detail view (except you'll only have two buttons)

Reply:
Added the additional "Campaign" line to the campaign detail view header, after `{{ object.title }}`.